code/game/gamemodes/gang/dominator.dm

```diff
@@ -48,6 +49,20 @@
 	if(gang && gang.is_dominating)
 		var/time_remaining = gang.domination_time_remaining()
 		if(time_remaining > 0)
+			var/open = 0
+			for(var/turf/T in circleviewturfs(center=user,radius=3))
+				if(!istype(T, /turf/closed))
```

Comment:
I can't check right now but what exactly does that check for? I could see it causing problems if its just a density check.